mycroft/messagebus/message.py

```diff
@@ -50,6 +101,20 @@ def reply(self, type, data, context={}):
         return Message(type, data, context=new_context)
 
     def publish(self, type, data, context={}):
+        """
+
+        Copy the original context and add passed in context.  Delete
```

Comment:
This comment actually helped me in that it called out this implementation is incorrect. The current implementation prevents users of publish from setting an explicit target. 

Again, the comments are more verbose than they need to be.